src/status_im/contexts/profile/contact/contact_request/style.cljs

```diff
@@ -7,4 +7,6 @@
 
 (def message-input-wrapper
   {:padding-vertical   8
+   :flex-shrink        1
+   :flex-direction     :column
```

Comment:
Is this needed? :eyes:

Reply:
I don't think `:flex-direction :column` is needed, I'll remove it 👍